m2e-maven-runtime/pom.xml

```diff
@@ -312,5 +312,78 @@
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<id>eclipse-sign-jnilibs</id>
```

Comment:
Shouldn't it be part of the regular "sign" profile that's already defined in parent?

Reply:
But this would activate the subsequent mac-signing for all projects and not only for those that have the `eclipse-sign-jnilibs.properties` in their root folder.

Reply:
ok, makes sense. It's a pity Maven doesn't allow condition executions.

Reply:
> ok, makes sense. It's a pity Maven doesn't allow condition executions.

You can open an enhancement request to maven :-)

Reply:
Yep, just opened https://issues.apache.org/jira/browse/MNG-7426

Reply:
> ok, makes sense. It's a pity Maven doesn't allow condition executions.

That would indeed makes thinks simpler. I also added the follow up #596 to not fail the build locally.
If conditional executions would be possible it could be handled in one profile.
When thinking about it again it would probably be simpler to make the execution of the ant-task conditional repetitively skip the signing procedure if `eclipse-sign-jnilibs.properties`.
But actually I hope that this is fixed in jansi and this work-around can be fixed soon.


> > ok, makes sense. It's a pity Maven doesn't allow condition executions.
> 
> You can open an enhancement request to maven :-)

I would support that. But I expect that such a feature was already requested many times (but I might be wrong).